app/models/category.rb

```diff
@@ -0,0 +1,9 @@
+class Category < ActiveRecord::Base
+  has_many :videos, -> { order "created_at DESC" }
```

Comment:
The other way how to write this would be using symbols notation `order(created_at: :desc)`